src/main/kotlin/uk/gov/justice/digital/hmpps/prisonregister/resource/PrisonResource.kt

```diff
@@ -128,7 +128,7 @@ class PrisonResource(private val prisonService: PrisonService, private val addre
     "/names",
     produces = [MediaType.APPLICATION_JSON_VALUE],
   )
-  @Operation(summary = "Get prison names", description = "prison id and full name")
+  @Operation(summary = "Get prison name/s", description = "prison id and full name")
```

Comment:
As I have used /names resource, i don't need to create a new endpoint+resource as parameters manage this new scenario.